src/events/AddReactionEventHandler.ts

```diff
@@ -1,8 +1,9 @@
-import { User, MessageReaction } from 'discord.js';
+import {User, MessageReaction, TextChannel} from 'discord.js';
 import BotConfig from '../BotConfig';
 import EventHandler from './EventHandler';
 import SelectRoleEventHandler from './roles/SelectRoleEventHandler';
 import ResolveRequstEventHandler from './requests/ResolveRequestEventHandler';
+import NewRequestEventHandler from "./requests/NewRequestEventHandler";
```

Comment:
Always use single quotes, because that's convention for some reason (also caught by `lint`)